docs/installation.rst

```diff
@@ -20,6 +20,35 @@ There are different ways to install |project_name|:
     have the latest code or use revision control (each release is
     tagged).
 
+.. _installation-requirements:
+
+Pre-Installation Considerations
+-------------------------------
+
+Repository Filesystem
+~~~~~~~~~~~~~~~~~~~~~
+
+- Filenames may contain any valid ascii character and be of any length. |project_name|
+  stores data only 3 directory levels deep and uses short directory and filenames.
+- |project_name| requires only read/write permissions within the repository filesystem.
```

Comment:
that sounds a bit strange. why "only"?
also, I guess we require "x" on the directories.

so, just say "... requires r/w access to the repository filesystem."